<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="my_commit_btn">
       <property name="text">
        <string>My Commit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="assigned_me_btn">
       <property name="text">
        <string>Assigned Me</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="user_name_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="user_name_label">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="product_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="product_label">
       <property name="text">
        <string>Product:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="product_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="project_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="project_label">
       <property name="text">
        <string>Project:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="project_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QLabel" name="module_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLabel" name="module_label">
       <property name="text">
        <string>Module:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QComboBox" name="module_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLabel" name="branch_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="10">
      <widget class="QLabel" name="branch_label">
       <property name="text">
        <string>Branch:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="11">
      <widget class="QComboBox" name="branch_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="type_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="type_label">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="type_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="severity_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="severity_label">
       <property name="text">
        <string>Severity:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QComboBox" name="severity_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="pri_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QLabel" name="pri_label">
       <property name="text">
        <string>Pri:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QComboBox" name="pri_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="9">
      <widget class="QLabel" name="version_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="10">
      <widget class="QLabel" name="version_label">
       <property name="text">
        <string>Version:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="11">
      <widget class="QLineEdit" name="version_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="os_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="os_label">
       <property name="text">
        <string>OS:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="os_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="browser_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="browser_label">
       <property name="text">
        <string>Browser:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QComboBox" name="browser_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QLabel" name="draft_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="draft_label">
       <property name="text">
        <string>Draft:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QComboBox" name="draft_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="template_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QLabel" name="template_label">
       <property name="text">
        <string>Template:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="11">
      <widget class="QComboBox" name="template_box">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="title_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="title_label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="title_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="keywords_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="keywords_label">
       <property name="text">
        <string>Keywords:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="keywords_edit"/>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPlainTextEdit" name="main_edit"/>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="mailto_icon">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="mailto_label">
       <property name="text">
        <string>Mailto:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="mailto_edit"/>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="save_to_draft_btn">
       <property name="text">
        <string>Save to Draft</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="save_to_template_btn">
       <property name="text">
        <string>Save to Template</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="commit_btn">
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
